cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -678,15 +678,31 @@ class HydratorPlusPlusTopPanelCtrl {
     };
     this.previewLoading = true;
     this.loadingLabel = 'Stopping';
-    this.stopTimer();
-    this.updateTimerLabelAndTitle();
     this.myPipelineApi
         .stopPreview(params, {})
         .$promise
         .then(
-          (err) => {
+          () => {
+            this.stopTimer();
+            this.updateTimerLabelAndTitle();
             this.previewLoading = false;
             this.previewRunning = false;
+            this.dataSrc.stopPoll(this.pollId);
+            this.pollId = null;
+
+            let pipelinePreviewPlaceholder = 'The preview of the pipeline';
+            let pipelineName = this.HydratorPlusPlusConfigStore.getName();
+            if (pipelineName.length > 0) {
+              pipelinePreviewPlaceholder += ` "${pipelineName}"`;
+            }
+            this.myAlertOnValium.show({
+              type: 'success',
```

Comment:
Could we define enum for status type if it also has other status such as `failure`?

Reply:
I believe these types are coming from an external library (ngStrap's alert) which we want to deprecate. I don't think it makes sense to add an enum for the status here, since the bigger to do is to replace this alert entirely.